solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LambdaMARTModel.java

```diff
@@ -193,14 +193,12 @@ public LambdaMARTModel(String name, List<Feature> features,
     final List<Object> jsonTrees = getParams().containsKey("trees") ?
         (List<Object>) getParams().get("trees") : null;
 
-    if ((jsonTrees == null) || jsonTrees.isEmpty()) {
-      throw new ModelException("LambdaMARTModel doesn't contain any trees");
-    }
-
-    for (final Object o : jsonTrees) {
-      final Map<String,Object> t = (Map<String,Object>) o;
-      final RegressionTree rt = new RegressionTree(t, fname2index);
-      trees.add(rt);
+    if (jsonTrees != null) {
```

Comment:
Yes, I think so. An empty jsonTrees corresponds to `trees = []` in the config whereas a null jsonTrees would correspond to the absence of any `trees = ...` in the config. Both logically are just 'missing' trees i.e. empty model, and later on the 'validate' functionality would flag up/reject both variants of an empty model if we choose that.